Detections/AWSCloudTrail/AWS_CredentialHijack.yaml

```diff
@@ -1,4 +1,5 @@
 id: c2f0bc12-3975-4b76-9b4a-6c056097297b
 name: Monitor AWS Credential abuse or hijacking
 description: |
-  'As part of content migration, this file is moved to new location. you can find here: https://github.com/Azure/Azure-Sentinel/blob/master/Solutions/Amazon Web Services'
\ No newline at end of file
+  'As part of content migration, this file is moved to new location. you can find here: https://github.com/Azure/Azure-Sentinel/blob/master/Solutions/Amazon%20Web%20Services/Analytic%20Rules/AWS_FullAdminPolicyAttachedToRolesUsersGroups.yaml'
```

Comment:
This file name is incorrect. Please correct it and also verify at all the places.